.github/workflows/tests_uv.yaml

```diff
@@ -1,11 +1,11 @@
-name: Tests
+name: Lumigator integration tests
 
 on:
   push:
 
 jobs:
   uv-example:
```

Comment:
I guess the name of the job should have been updated, now that I notice? Same in other workflows.

Reply:
Yeah you're right, we should update them to be accurate. 😄